pkg/infra/process/root_check_windows.go

```diff
@@ -0,0 +1,35 @@
+// +build windows
+
+package process
+
+import (
+	"fmt"
+
+	"golang.org/x/sys/windows"
+)
+
+func isServerRunningAsRoot() (bool, error) {
+	var sid *windows.SID
+	defer windows.FreeSid(sid)
+
+	//  The Go API for this is a direct wrap of the official C++ API.
+	// See https://docs.microsoft.com/en-us/windows/desktop/api/securitybaseapi/nf-securitybaseapi-checktokenmembership
+	err := windows.AllocateAndInitializeSid(
```

Comment:
:eyes:

Reply:
If the windows check feels uncertain I think we should ship the Linux check regardless. It's by far the most popular OS anyway :)

Reply:
True that. I'm happy enough to do that for the time being

Reply:
Sounds like a plan :+1:  I tried to test this basically running `sudo make run` but it just stall guessing go environment for root is not setup. But I don't get any errors at least.

Reply:
Cool. In that case I'll remove the Windows part and play around a bit more with the linux check and see what comes out